vsts/pipelines/_signbinary.yml

```diff
@@ -5,29 +5,34 @@ steps:
     ignoreDirectories: '$(Build.SourcesDirectory)/tests'
 
 - powershell: |
-   Write-Host "##vso[task.setvariable variable=SignType]real"
+   Write-Host "##vso[task.setvariable variable=SignType;isOutput=true]real"
+  name: setSignTypeVariable
   displayName: 'Set sign binary variable'
   condition: >
    and(in(variables['Build.Reason'], 'Schedule', 'Manual'), eq(variables['Build.DefinitionName'], 'Oryx-CI'), 
    or(startsWith(variables['Build.SourceBranch'], 'refs/heads/master'), startsWith(variables['Build.SourceBranch'],'refs/heads/patch/' )))
 
+- script: |
+    echo $(setSignTypeVariable.SignType)
+  name: SignType
+
 - task: NuGetToolInstaller@0
   displayName: 'Use NuGet 4.6.2'
   inputs:
     versionSpec: 4.6.2
-  condition: and(succeeded(), eq(variables['SignType'], 'real'))
+  condition: and(succeeded(), eq(variables['setSignTypeVariable.SignType'], 'real'))
```

Comment:
Didn't we have a test to check if the dll was actually signed, case in which we add a .txt file to the build output that is then checked in the release definition?

Reply:
that txt file was for integration test ran successfully